content/en/docs/tasks/administer-cluster/unknown-version-interoperability-proxy.md

```diff
@@ -0,0 +1,47 @@
+---
+reviewers:
+- jpbetz
+title: Unknown Version Interoperability Proxy Alpha
+content_type: concept
+weight: 220
+---
+
+<!-- overview -->
+This page provides an overview of a new capability added to kube-apiservers that enables them to proxy a resource request to the correct peer kube-apiserver when there are multiple apiservers at varied versions in a cluster.
+
+### Unknown Version Interoperability Proxy Alphs
+
+{{< feature-state state="alpha"  for_k8s_version="v1.28" >}}
+
+The Unknown Version Interoperability Proxy feature is controlled by a
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+and is not enabled by default. You can activate Unknown Version Interoperability Proxy by enabling
+the feature gate named `UnknownVersionInteroperabilityProxy` when you start the
+{{< glossary_tooltip text="API Server" term_id="kube-apiserver" >}}:
+
+```shell
+kube-apiserver \
+--feature-gates=UnknownVersionInteroperabilityProxy=true \
+# required flag for this feature
+--peer-ca-file=`ca bundle to verify serving certs of kube-apiserver`
+# optional flags for this feature
+--peer-advertise-ip=`ip of this kube-apiserver that should be used by peers to proxy requests`
+--peer-advertise-port=`port of this kube-apiserver that should be used by peers to proxy requests`
+# …and other flags as usual
+```
+
+To support correct handling of resource requests that are made when a cluster has multiple kube-apiservers at varied versions, a new filter is added to the handler chain in the aggregator which proxies clients to an apiserver that is capable of handling their request.
+
+When a kube-apiserver receives a resource request, it first checks which apiservers can serve the requested resource using StorageVersion API. If the resource is known to the local apiserver, the request is handled locally.
```

Comment:
```suggestion

## Interoperability proxying

When you enable interoperability proxying, the [aggregation layer](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) 
loads a special filter that identifies API groups / versions / kinds that the local server doesn't recognise,
and tries to proxy those requests to a peer API server that is capable of handling the request.

When a kube-apiserver receives a resource request, it first checks which API servers can serve the
requested resource. The check happens using the internal `StorageVersion` API.
If the resource is known to the local API server, the request is handled locally. If a configured APIService
specifies proxying to an extension API server, that proxying happens following the usual
[flow](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer/) for
extension APIs.
```

Reply:
Updated with some amends/details about proxying to extension API server